platforms/Linux/centos/8/Dockerfile

```diff
@@ -22,9 +22,6 @@ RUN yum-builddep -y /root/rpmbuild/SPECS/swift-lang.spec
 # Get the sources for Swift as defined in the spec file
 RUN spectool -g -R /root/rpmbuild/SPECS/swift-lang.spec
 
-# Add the patches
-ADD patches/*.patch /root/rpmbuild/SOURCES/
```

Comment:
keep this in case we have patches in the future

Reply:
We should avoid patches, and have this resolved before release.

Reply:
sure we can add it back if we need it